.github/workflows/lint.yaml

```diff
@@ -4,39 +4,17 @@ on: [ push, pull_request ]
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-versions: [ 1.16.x, 1.17.x, 1.18.x, 1.19.x ]
```

Comment:
For lint, you can get rid of this axis in the matrix, and remove the "install" step